adaptive_scheduler/kernel_mappings.py

```diff
@@ -54,9 +52,6 @@
 multi_rg_log = logging.getLogger('rg_logger')
 rg_log = RequestGroupLogger(multi_rg_log)
 
-redis = Redis.from_url(url=os.getenv('REDIS_URL', 'redis://redis'), socket_connect_timeout=15,
```

Comment:
I moved the global redis instance into the models.py file. I'm not sure if its okay to have multiple global redis instances, or if there is any benefit to doing it one way or another...